### DISCUSSION TITLE
VM-Fi聲麥-即時AI語音翻譯

### DISCUSSION THREAD

**Discussion by bonny123865** [1101-GS3073-C/Python-Community-GS3073-C#155] [General]
**新聞內容:** 
來自花蓮的彭德新、彭德元兄弟檔創設VM-Fi聲麥無線有限公司，花了兩年多，成功研發一套AI人工智慧與結合5G高速通訊技術的即時翻譯系統，「VM-Fi聲麥無線-5G即時AI語音翻譯解決方案」，透過5G即時AI語音翻譯僅花2秒多，和一般人工翻譯需要10多秒，快了5倍，成效高超。美國消費者技術協會近日公布今年度CES 2022創新獎得獎名單，這套系統就獲得智慧城市創新獎。
可達到台上一人即講、台下百人即聽的即時語音轉譯，「VM-Fi聲麥無線」會持續提供跨越語言、距離的商業通訊方案，實現語言無障礙的智慧城市生活。
**新聞心得:**
這個即時翻譯的處理時間好快，讓接收者對於聽取的內容時間間隔不會太長，應用在展覽解說或是跟外商商談時都很適合，而且50公尺內都能接收到應該算不錯了！ !
**新聞來源:**
https://udn.com/news/story/7240/5986661

![163402279620827_P7659618](https://user-images.githubusercontent.com/90883526/147400782-9f282c01-af51-4400-ad42-14ba2968f088.jpg)

**Comment by pocper1** [comment]
未來有機會與外國人即時翻譯，馬上了解外國人再想什麼嗎~

**Comment by briankuo123** [comment]
有沒有可能我拿去上我現在的課，它也可以為我翻譯教授到底在講什麼

**Comment by qeeeee39** [comment]
實現了多拉A夢的翻譯蒟蒻!!!

**Comment by wh20502** [comment]
有時候我會想，像是日文的動詞在最後，那麼AI若想發展及時翻譯，後面的句子可能是要推測囉？

**Comment by jack90325** [comment]
這種翻譯系統也太強，那這樣又要有人類被機器取代了

**Comment by Shiba010** [comment]
這樣人人都有即席翻譯的助理了！
